lib/main/atom/tooltipManager.ts

```diff
@@ -15,10 +15,19 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any>el).querySelectorAll(selector)
```

Comment:
I was pointed in #1287 that here it should use `querySelector` since we only care about the first element anyway, line 19 should of course be changed to `return $(found)`